### PR TITLE
feat: Using sqlite CGO is for building a 32-bit Windows operating system

### PR DESCRIPTION
**Pull request by horizon220222**
#### Why do we need to use github.com/gogf/gf/contrib/drivers/sqlitecgo/v2?

Using sqlite CGO is for building a 32-bit Windows operating system

https://github.com/glebarez/go-sqlite does not support building a  32-bit Windows operating system.

#### sqlite

【using CGO】github.com/mattn/go-sqlite3
【not using CGO】https://github.com/glebarez/go-sqlite

#### goframe driver

[using CGO]github.com/gogf/gf/contrib/drivers/sqlitecgo/v2
[not using CGO]github.com/gogf/gf/contrib/drivers/sqlite/v2

### CODE REVIEW

**Comment by horizon220222** [comment]
You need to set the environment variable CGO_ENABLED=1 and make sure that GCC is installed on your path.

windows: https://jmeubank.github.io/tdm-gcc/

**Comment by gqcn** [comment]
@horizon220222 CI失败了，可以参考已有的CI脚本是怎么写的 https://github.com/gogf/gf/blob/master/.github/workflows/build_and_test.sh
image

**Comment by horizon220222** [comment]
I will take care of it over the weekend.

**Comment by horizon220222** [comment]
fix: building errors 1.15 and 1.16 (undefined: math.MaxInt)

**Review by gqcn — CHANGES_REQUESTED** [review]

**Comment by horizon220222** [comment]
最近比较忙，已经修改

File: contrib/drivers/sqlitecgo/sqlite.go [hunk 1]
```diff
@@ -0,0 +1,253 @@
+// Copyright GoFrame Author(https://goframe.org). All Rights Reserved.
+//
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file,
+// You can obtain one at https://github.com/gogf/gf.
+
+// It does not support Save/Replace features.
+
+package sqlitecgo
```
Comment (gqcn) [inline contrib/drivers/sqlitecgo/sqlite.go:9]: 缺少包注释描述，可以参考下其他`driver`包怎么写的包注释描述哈。

**Review by gqcn — CHANGES_REQUESTED** [review]

File: contrib/drivers/sqlitecgo/sqlite.go [hunk 2]
```diff
@@ -0,0 +1,256 @@
+// Copyright GoFrame Author(https://goframe.org). All Rights Reserved.
+//
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file,
+// You can obtain one at https://github.com/gogf/gf.
+
+// Package sqlitecgo implements gdb.Driver, which supports operations for database SQLite.
+//
+// Note:
+// 1. It needs manually import: _ "https://github.com/mattn/go-sglite3"
```
[the remark this answers is not in the document]
Comment (gqcn) [reply contrib/drivers/sqlitecgo/sqlite.go]: @horizon220222 只是去掉这一行的注释，因为注释是错误的。

File: contrib/drivers/sqlitecgo/sqlite.go (hunk 1 above)
Comment (horizon220222) [reply contrib/drivers/sqlitecgo/sqlite.go:12]: 哈哈哈 注释确实没写啊 我补充下

**Review by gqcn — APPROVED** [review]

**Pull request merged**